conda_smithy/cli.py

```diff
@@ -184,6 +184,8 @@ def __call__(self, args):
         try:
             configure_feedstock.main(args.feedstock_directory)
             print("\nCI support files regenerated. These need to be pushed to github!")
+            print("\nYou can commit the changes with:\n\n"
+                  "    git commit -am 'MNT: rerender with conda-smithy %s'" % __version__)
```

Comment:
Maybe add `version` before.

Also how many characters does this end up running roughly?